lib/src/main/java/uk/gov/di/ipv/core/library/domain/gpg45/Gpg45Scores.java

```diff
@@ -142,6 +143,30 @@ private List<Gpg45Scores.Evidence> diffEvidence(Gpg45Scores target) {
         return evidenceDiff;
     }
 
+    public Gpg45Scores minimumRequiredToMeetProfileGivenCurrentFraudScore(
```

Comment:
My only suggestion would be to maybe replace `RequiredToMeetProfile` with something shorter as you can infer some of this from the fact we have a `profiles` argument